projects/transformers/experiments/ablations.py

```diff
@@ -291,6 +290,42 @@ def pct_start_hp_space(trial):
     model_name_or_path="/mnt/efs/results/pretrained-models/transformers-local/bert_sparse_80%_kd_onecycle_lr_100k",  # noqa: E501
 )
 
+#############
+# Deepspeed #
+#############
+
+# This lr-range test is based on `bert_sparse_100k_kd_lr_range_test` and adapted
+# for deepspeed training.
+# With this test the best `max_lr` value found is `0.0017`.
```

Comment:
Didn't the max_lr of 0.0012 cause a nan loss? I wonder why this test is suggesting something higher.

Reply:
the nan loss was fixed with `"gradient_clipping": 1.0,`